libs/utils/utils.js

```diff
@@ -1322,8 +1322,9 @@ export async function loadArea(area = document) {
 
   if (allIcons.length) {
     const { default: loadIcons, decorateIcons } = await import('../features/icons/icons.js');
-    await decorateIcons(area, allIcons, config);
-    await loadIcons(allIcons);
+    const areaIcons = area.querySelectorAll('span.icon');
+    await decorateIcons(area, areaIcons, config);
+    await loadIcons(areaIcons);
```

Comment:
Do we need to "duplicate" this or could we bring L1306-1309 in here and just run `querySelectorAll` once?

Reply:
I believe the code in L1306-1309 is there to address CLS concerns.

Reply:
Yep I just verified that when I move it there is CLS. Not sure if there is another way to address that.